docs/en/config/actuators.md

```diff
@@ -575,6 +575,28 @@ Another way to test without using the sliders would be to set the [`COM_PREARM_M
 
 :::
 
+##### Split Servo Deflections
```

Comment:
Mention that this method is only applicable to PWM outputs. E.g. UAVCAN servos don't have the trim exposed, do they?